cache-config/t3c-apply/config/config.go

```diff
@@ -329,25 +328,25 @@ func GetCfg() (Cfg, error) {
 	var tsHome = ""
 	if *tsHomePtr != "" {
 		tsHome = *tsHomePtr
-		fmt.Printf("set TSHome from command line: '%s'\n\n", TSHome)
+		log.Infof("set TSHome from command line: '%s'\n\n", TSHome)
 	}
 	if *tsHomePtr == "" { // evironment or rpm check.
 		tsHome = os.Getenv("TS_HOME") // check for the environment variable.
 		if tsHome != "" {
-			fmt.Printf("set TSHome from TS_HOME environment variable '%s'\n", TSHome)
+			log.Infof("set TSHome from TS_HOME environment variable '%s'\n", TSHome)
 		} else { // finally check using the config file listing from the rpm package.
 			tsHome = GetTSPackageHome()
 			if tsHome != "" {
-				fmt.Printf("set TSHome from the RPM config file  list '%s'\n", tsHome)
+				log.Infof("set TSHome from the RPM config file  list '%s'\n", tsHome)
 			} else {
-				fmt.Printf("no override for TSHome was found, using the configured default: '%s'\n", TSHome)
+				log.Infof("no override for TSHome was found, using the configured default: '%s'\n", TSHome)
 			}
 		}
 	}
 	if tsHome != "" {
 		TSHome = tsHome
 		TSConfigDir = tsHome + "/etc/trafficserver"
-		fmt.Printf("TSHome: %s, TSConfigDir: %s\n", TSHome, TSConfigDir)
+		log.Infof("TSHome: %s, TSConfigDir: %s\n", TSHome, TSConfigDir)
```

Comment:
All of these logs happen before the `log.InitCfg(cfg)` call below. Until that's called, the loggers don't exist. So these logs will never be written.

The easiest solution here is probably to put the strings in an array, i.e.
```
toInfoLog := []string{}
```
And then e.g.
```
	if *tsHomePtr != "" {
		tsHome = *tsHomePtr
		toInfoLog = append(toInfoLog, fmt.Sprintf("set TSHome from command line: '%s'", TSHome)
	}
```

And then, after the loggers are initialized:

```
	if err = log.InitCfg(cfg); err != nil {
		return Cfg{}, errors.New("Initializing loggers: " + err.Error() + "\n")
	}

	for msg := range toInfoLog {
		log.Infoln(msg)
	}
```

Reply:
Added array for info messages and for loop to print them out once logging is initialized.